src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,39 @@
+.nav {
+  &__list {
+    display: flex;
+    list-style: none;
+  }
+
+  &__link {
+    display: inline-block;
+    text-decoration: none;
+    color: $link-color;
+    text-transform: uppercase;
+    font-weight: $regular-weight;
+    font-size: 12px;
+    padding: 23px 0;
+    margin-left: 20px;
+    transition: color 0.4s;
```

Comment:
Better to use same duration for all transitions on the page (you also created variable for this purpose)